mod.d/mysqldlog.yaml

```diff
@@ -44,7 +44,14 @@ content: !!str |
       cp -r /var/log/mysqld.log* $EC2RL_GATHEREDDIR/mysqldlog
       echo "I have attempted to copy the /var/log/mysqld.log* files to ${EC2RL_GATHEREDDIR}/mysqldlog"
   else
-      echo "No /var/log/mysqld.log* file available or permission denied"
+      echo "No /var/log/mysqld.log* files available or permission denied"
+  fi
+  if [ -r /var/log/mariadb ]; then
+      mkdir -p $EC2RL_GATHEREDDIR/mysqldlog/mariadb
```

Comment:
I like putting the MariaDB logs in their own subdirectory, but we should do the same for MySQL as well (and the same would go for any other drop in MySQL replacement in the future).